### PR TITLE
Eslint airbnb

### PR DESCRIPTION
**Pull request by kalmanb**
- move folders to `src`
- add airbnb eslint config
- update formatting

Still a number of errors to work through

### CODE REVIEW

**Comment by kalmanb** [comment]
@qisaw @d4ncer have a look at the airbnb style - see what you think. You can run `npm run lint` or `npm run format` 

Sorry after moving the files couldn't keep a proper diff.

File: package.json [hunk 1]
```diff
@@ -4,11 +4,14 @@
   "description": "Movio's TodoMVC stack",
   "main": "index.js",
   "scripts": {
-    "start": "webpack-dev-server --hot --inline --progress --colors",
+    "start": "webpack-dev-server --hot --inline --progress --colors --host 0.0.0.0",
     "build": "webpack --progress --colors",
     "test": "standard-format -w",
-    "lint": "eslint",
-    "format": "eslint"
+    "lint": "eslint src",
```
Comment (qisaw) [inline package.json:10]: minor but we should probably add these commands into the readme

File: package.json [hunk 2]
```diff
@@ -37,6 +40,11 @@
     "babel-preset-movio": "^1.0.2",
     "babel-runtime": "^6.6.1",
     "css-loader": "^0.23.1",
+    "eslint": "^2.8.0",
+    "eslint-config-airbnb": "^7.0.0",
+    "eslint-plugin-import": "^1.5.0",
+    "eslint-plugin-jsx-a11y": "^0.6.2",
+    "eslint-plugin-react": "^5.0.1",
```
Comment (qisaw) [inline package.json]: think this should be a peer of 4.3
`WARN eslint-config-airbnb@7.0.0 requires a peer of eslint-plugin-react@^4.3.0 but none was installed.`

File: package.json (hunk 1 above)
Comment (qisaw) [inline package.json:10]:
```
npm run format
npm start
```

returns a syntax error
`Module build failed: SyntaxError: /Users/wasiqk/work/movio-todomvc/src/todos/components/TodoItem.js: Unexpected token (2:4)`

File: package.json [hunk 3]
```diff
@@ -4,11 +4,14 @@
...
-    "start": "webpack-dev-server --hot --inline --progress --colors",
+    "start": "webpack-dev-server --hot --inline --progress --colors --host 0.0.0.0",
     "build": "webpack --progress --colors",
     "test": "standard-format -w",
-    "lint": "eslint",
-    "format": "eslint"
+    "lint": "eslint src",
+    "format": "eslint src --fix"
+  },
+  "eslintConfig": {
```
Comment (d4ncer) [inline package.json]: Can we move all eslint config into `.eslintrc` to avoid polluting `package.json`?

File: package.json (hunk 1 above)
Comment (kalmanb) [inline package.json:10]: Yep to readme. We can also make it a loader in webpack in the future

File: package.json (hunk 2 above)
Comment (kalmanb) [inline package.json]: So we should back this down to 4.3?

File: package.json (hunk 3 above)
Comment (kalmanb) [inline package.json]: Sure can. I prefer it in the package - I have no idea why.
Do we prefer these kind of configs, eslint, babel etc in dot files or package? (I think I know the answer ...)

**Comment by kalmanb** [comment]
Fixed up spread operator issues and moved eslint config to .eslintrc
cc @qisaw fix was simple see .eslintrc

File: package.json (hunk 2 above)
Comment (qisaw) [inline package.json]: Yep until eslint-config-airbnb update their dependencies

**Comment by qisaw** [comment]
Nesting the elements was causing the formatting to break, moving this into a variable fixed the issue.

**Pull request merged**